crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -597,8 +597,13 @@ pub struct StandardMaterialUniform {
     pub emissive: Vec4,
     /// Color white light takes after travelling through the attenuation distance underneath the material surface
     pub attenuation_color: Vec4,
-    /// The transform applied to the UVs corresponding to ATTRIBUTE_UV_0 on the mesh before sampling. Default is identity.
-    pub uv_transform: Mat3,
+    /// The x-axis of the mat2 of the transform applied to the UVs corresponding to ATTRIBUTE_UV_0 on the mesh before sampling. Default is [1, 0].
+    pub uv_transform_x_axis: Vec2,
+    /// The y-axis of the mat2 of the transform applied to the UVs corresponding to ATTRIBUTE_UV_0 on the mesh before sampling. Default is [0, 1].
+    pub uv_transform_y_axis: Vec2,
+    /// The translation of the transform applied to the UVs corresponding to ATTRIBUTE_UV_0 on the mesh before sampling. Default is [0, 0].
+    pub uv_transform_translation: Vec2,
+    pub padding: Vec2,
```

Comment:
Some documentation that this is specific to WebGL2 (and apparently DX12) would be useful. If we ever migrate to a WebGPU-only support mode, this might otherwise be difficult to find.

Reply:
There are already places in Bevy using the alternative name `wasm_padding` for this purpose, so there is a precedent for that name implicitly meaning "WebGL is annoying, plz remove this when possible"

Reply:
this one is just for DX12, works without it on WebGL2

Reply:
@mockersf TIL DX12 also requires padding